composables/useUser/index.ts

```diff
@@ -59,8 +59,8 @@ export const useUser = () => {
           accessToken: account.accessToken,
           accessTokenExpiration: account.accessTokenExpiration,
           refreshToken: account.refreshToken,
-          refreshTokenExpiration: account.refreshToken,
-          userId: account?.refreshToken,
+          refreshTokenExpiration: account.refreshTokenExpiration,
```

Comment:
use account?.refreshTokenExpiration instead of account.refreshTokenExpiration

Reply:
fixed.